README.md

```diff
@@ -201,7 +209,7 @@ Revocations, (partial) dispensations and cancellations contain the following dat
 <p>
 The following event types exist:<p>
 <p>
-<code>revoke | partial_dispense | full_dispense | cancel</code><p>
+<code>revoke | partial_dispense | full_dispense | cancel | lock | unlock</code><p>
```

Comment:
Where is the reason stored?

Reply:
The reason is stored in both logs.

Reply:
Shouldn't we include it in this table?

Reply:
See https://github.com/certifaction/hinsign-cli/pull/15/commits/3f0c3fda46e4a4ad6bb456486578e7669981cad6.